src/cpp/session/modules/SessionPackages.R

```diff
@@ -761,6 +761,53 @@ if (identical(as.character(Sys.info()["sysname"]), "Darwin") &&
       cat(helloWorldRd,
           file = file.path(packageDirectory, "man", "hello.Rd"),
           sep = "\n")
+      
+      if (usingRcpp)
+      {
+         ## Ensure 'src/' directory exists
+         if (!file.exists(file.path(packageDirectory, "src")))
+            dir.create(file.path(packageDirectory, "src"))
+         
+         ## Write a 'hello world' for C++
+         helloWorldCpp <- c(
+            "#include <Rcpp.h>",
+            "using namespace Rcpp;",
+            "",
+            "// This is a simple function using Rcpp that",
+            "// creates an R list containing a character vector",
+            "// c(\"foo\", \"bar\") and a numeric vector",
+            "// c(0, 1).",
+            "",
+            "// The comment '// [[Rcpp::export' immediately above the function",
+            "// signals that this function should automatically be exported to",
+            "// R when the package is built.",
+            "//",
```

Comment:
Let's separate the explanatory comments from the Rcpp::export comment just for clarity